### PR TITLE
Ana/disable validator picture update notification

### LINKED ISSUE #5074: notifications: issue with validator profile image updates
**Describe the bug**

- this notification should only be triggered when there is an on-chain update (not a script runner update)
- for some reason these are duplicated

Screen Shot 2020-10-07 at 3 47 58 PM

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #5074

**Description:**

Right now fixes validator picture update notifications when there hasn't been any real update.

But it doesn't fix those duplicates yet. I can fix it in this one too though

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: api/lib/stores/block-store.js [hunk 1]
```diff
@@ -102,10 +102,11 @@ class BlockStore {
   // this adds all the validator addresses to the database so we can easily check in the database which ones have an image and which ones don't
   async updateDBValidatorProfiles(validators) {
     const validatorRows = Object.values(validators).map(
-      ({ operatorAddress, name, identity }) => ({
+      ({ operatorAddress, name, identity, picture }) => ({
         operator_address: operatorAddress,
         name,
-        profile_identifier: identity
+        profile_identifier: identity,
+        picture
```
Comment (faboweb) [inline api/lib/stores/block-store.js:109]: why is this fixing it?

Comment (faboweb) [reply api/lib/stores/block-store.js:109]: Wouldn't this just overwrite the pictures in the db?

**Comment by jbibla** [comment]
honestly, this is such a low priority alert it might be worth just removing it altogether @faboweb

File: api/lib/stores/block-store.js (hunk 1 above)
Comment (Bitcoinera) [reply api/lib/stores/block-store.js:109]: https://github.com/luniehq/lunie/issues/5074#issuecomment-707924607

Comment (Bitcoinera) [reply api/lib/stores/block-store.js:109]: Yes, it overwrites `null`s and outdated pictures with the current ones, which is good

**Comment by faboweb** [comment]
let's remove it

**Comment by Bitcoinera** [comment]
> let's remove it

Alright, so I rename this to "remove the validator picture update"

**Review by faboweb — APPROVED** [review]

**Pull request merged**